package.json

```diff
@@ -73,6 +73,13 @@
       ],
       "order": 4
     },
+    "alternateCompletion": {
+      "title": "Alternate Completion Keybindings",
+      "description": "If this setting is true, enter will only confirm the selected suggestion if it has been changed by the user (e.g. by using the movement keys).",
+      "type": "boolean",
+      "default": false,
```

Comment:
We could reasonably have this be the default, but that's probably just a matter of taste and I didn't want to change the default behaviour. Opinions?